packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card-banner.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import RootStore from 'Stores/index';
+import { connect } from 'Stores/connect';
+import { TVerificationStatusBannerProps } from '../props.types';
+import { getAuthenticationStatusInfo } from '@deriv/shared';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+const VerificationStatusBanner = ({
+    account_status,
+    account_type,
+    card_classname,
+    disabled,
+    is_virtual,
+    type_of_card,
+    should_restrict_bvi_account_creation,
+}: TVerificationStatusBannerProps) => {
+    const {
+        poi_not_submitted_for_vanuatu,
+        poi_or_poa_not_submitted,
+        poi_verified_for_vanuatu,
+        poi_verified_for_bvi_labuan_maltainvest,
+        poi_pending_for_bvi_labuan_maltainvest,
+        poi_pending_for_vanuatu,
+        poi_resubmit_for_vanuatu,
+        poi_resubmit_for_bvi_labuan_maltainvest,
+        poi_poa_verified_for_bvi_labuan_maltainvest,
+        poi_acknowledged_for_bvi_labuan_maltainvest,
+        poa_acknowledged,
+        need_poa_resubmission,
+        need_poi_for_bvi_labuan_maltainvest,
+        poa_pending,
+    } = getAuthenticationStatusInfo(account_status);
+
+    const getAccountTitle = () => account_type && (account_type === 'synthetic' ? 'Synthetics' : 'Financial');
+
+    const is_svg = type_of_card && type_of_card === 'svg';
+    const is_vanuatu = type_of_card && type_of_card === 'vanuatu';
+    const is_bvi = type_of_card && type_of_card === 'bvi';
```

Comment:
```suggestion
    const is_svg = type_of_card === 'svg';
    const is_vanuatu = type_of_card === 'vanuatu';
    const is_bvi = type_of_card === 'bvi';
```
@amina-deriv could you please check if it's gonna work the same way?🙂

Reply:
@maryia-binary it will work , But in my opinion it is better to test if type_of_card is not undefined.

Reply:
@amina-deriv i guess i get what you strive to achieve: checking for undefined here instead of writing `if (is_virtual && type_of_card && is_svg) {`
alright then🙂